modules/core/src/v2/coins/algo.ts

```diff
@@ -135,12 +135,7 @@ export class Algo extends BaseCoin {
    * @returns {boolean} True if okay to send 0 value, false otherwise
    */
   valuelessTransferAllowed(type?: unknown): boolean {
```

Comment:
please remove the `type` param from here and from the base class - I think it was only added for this use case which we don't need any more.